Scripts/parameters/demand/hb_business.json

```diff
@@ -95,10 +95,19 @@
                 "time": -0.04512
             },
             "log": {
-                "attraction_size": 1
+                "attraction_size": 1,
+                "transform": -0.1
             },
             "attraction_size": {
                 "workplaces": 1
+            },
+            "transform": {
+                "attraction": {
+                    "cost*within_zone": 1
+                },
+                "impedance": {
+                    "cost": 1
+                }
```

Comment:
There is no log-cost parameter in the non-home-based models, so we should not add the transformation in them.